modular_skyrat/master_files/code/modules/antagonists/traitor/objectives/kill_pet.dm

```diff
@@ -41,7 +41,7 @@
 	if(istype(target_pet, /mob/living/simple_animal/pet/dog/corgi/borgi))
 		name = "Emag the roboticist's most-prized borgi, E-N"
 		description = "We received an untracable contract from someone addressed 'ianfan2489@solfed.nt', they want the robotic clone of Ian dead- and it has to be dramatic. We will supply you with a cryptographic sequencer from our basic loadout options to shortcircuit E-N's safety, it will give you a single attempt to use it. Oh, and remember to run like hell when you do... "
-		telecrystal_reward = 3 // Because these are more complicated than 'kill the pet', the reward is bigger
+		telecrystal_reward = 2 // Because these are more complicated than 'kill the pet', the reward is bigger
```

Comment:
I recall you mentioning you wanted to make this harder to do, with a do_after so people couldn't do drive by emags? I think that would be more fitting here. also I feel like if the reward is going down the TC penalty for failure should too

Reply:
True! I still think it should remain 2 TC anyways. I changed the penalty to 2 TC as well.